modules/020-deckhouse/hooks/generate_selfsigned_ca.go

```diff
@@ -20,97 +20,38 @@ import (
 	"fmt"
```

Comment:
Because we work here, we can rename hook name, for example, to `generate_webhook_tls_certificate`. When i look at `generate_selfsigned_ca` name i do not understand what hook will do.

Reply:
Makes sense